### ISSUE TITLE
Analyze data on e-government across countries

### ISSUE THREAD

**Issue by pluralitybook** [pluralitybook/plurality#15]
Add to the Introduction both text and a figure highlighting international comparisons on e-government development across regime types, possibly including some light regression analysis.  A possible source is the UN's E-Government Development Index.

**Comment by Jordanu12** [comment]
I looked into this using the UN E-Gov Development  EDGI score and Freedom House Regime Typology of: Free, Partially Free and Not Free.  https://publicadministration.un.org/egovkb/Data-Center  I asked Bing to do some crude analysis here are the findings:

- Free countries tend to have higher EGDI scores than partly free or not free countries in all regions, except for MENA where the difference is not significant.
- Very high EGDI scores are mostly found in free countries, especially in Europe and Asia-Pacific. No not free country has a very high EGDI score.
- Low EGDI scores are mostly found in not free countries, especially in Africa. No free country has a low EGDI score.
- Partly free and not free countries have similar EGDI scores in some regions, such as Americas and MENA, but not in others, such as Africa and Asia-Pacific.

 I also explored correcting for GDP Per Capita, which seems to correlate with e-government development, even when their economic development is taken into account, this doesn't seem to make a difference.
 
 **However, there are specific, striking example you can cite of authoritarian regimes scoring higher than democratic governments:
 UAE, is 13th, Kazakhstan is 28th, and Saudia Arabia is 31, ahead of:
Canada 32, Italy 37, Brazil 49, and Mexico 62**

 This data can be cited as: United Nations Department of Economic and Social Affairs. E-Government Knowledge Database, 2022. https://publicadministration.un.org/egovkb/Data-Center.

**Comment by pluralitybook** [comment]
I think a key question here is controlling for level of development.  Free countries are richer than unfree ones.  My guess is controlling for income, unfree ones are higher EDGI.

**Comment by Jordanu12** [comment]
Right, what I thought as well, but per above "I also explored correcting for GDP Per Capita, which correlates with e-government development; however, when economic development is taken into account and corrected for, this doesn't seem to make a difference. There is still a correlation between democracy and EGDI index.

**Comment by ccerv1** [comment]
OK, this was a cool dataset! I went with @Jordanu12 ideas and came up with two versions. I think the second one is more interesting.

![egov_hists](https://github.com/pluralitybook/plurality/assets/42869436/c55ad1ee-01a1-49cb-860f-45f00e0ed0d1)

![egov_facets](https://github.com/pluralitybook/plurality/assets/42869436/5714aefc-43b1-45af-9b88-df47c9fdfcd6)

**Comment by audreyt** [comment]
The Introduction now has the requested cross-country analysis: 2-0 compares e-government development across regime types using the UN EGDI ([^EGDI]). Closing as completed.
